component/component/provider.jsonnet

```diff
@@ -215,6 +215,11 @@ local controllerConfigRef(config) =
           resources: [ 'providerconfigs' ],
           verbs: [ 'get', 'list', 'watch', 'update', 'patch', 'create', 'delete' ],
         },
+        {
```

Comment:
This seems necessary for the new SLI exporter, will remove it once the new version is released

Reply:
I switched to xobjectbucket so it seems superfluous now

Reply:
do we need all the verbs though?

Reply:
As @wejdross says we won't need them at all. I'll remove them.